src/gobstuf/rest/brp/base_view.py

```diff
@@ -252,8 +253,10 @@ def _transform_query_parameter_value(self, value: str):
         :param value:
         :return:
         """
-        if not value:
+        if value is None or value == '':
             return None
```

Comment:
if value in [None, '']

Reply:
or even better if value == '' return None, because None value is already catched in the elif

Reply:
Good one. Moved the isinstance(value, str) check up, because otherwise the logic was a bit weird to understand (checking for "' first).